### PR TITLE
Added new reformat commit to .git-blame-ignore-revs

### PR DESCRIPTION
**Pull request by mkmkme**

### CODE REVIEW

File: .git-blame-ignore-revs [hunk 1]
```diff
@@ -12,3 +12,6 @@ c41dd77a3e93e02be3c4bc75d8c76b7b4169a4ce
 
 # Removed terms `master` and `slave` from the source code (#591)
 54c97479356ecf41b4b63733494a1be2ab919e17
+
+# Set ColumnLimit to 0 and reformat
```
Comment (enjoy-binbin) [inline .git-blame-ignore-revs]:
```suggestion
# Set ColumnLimit to 0 and reformat (#1045)
```

**Comment by mkmkme** [comment]
True! Thanks :)

File: .git-blame-ignore-revs [hunk 2]
```diff
@@ -12,3 +12,6 @@ c41dd77a3e93e02be3c4bc75d8c76b7b4169a4ce
 
 # Removed terms `master` and `slave` from the source code (#591)
 54c97479356ecf41b4b63733494a1be2ab919e17
+
+# Set ColumnLimit to 0 and reformat (#1045)
+876850385aaf4ca3a1bcb42f4445b60d899243b3
```
Comment (enjoy-binbin) [inline .git-blame-ignore-revs]: 876850385aaf4ca3a1bcb42f4445b60d899243b3
af811748e7819a5ac31a6df4b21622aa58c64ae4
this is the right sha?


```suggestion
af811748e7819a5ac31a6df4b21622aa58c64ae4
```

Comment (mkmkme) [reply .git-blame-ignore-revs]: Oh! Sorry, github tricked me into passing my fork's commit hash :facepalm: 

Thanks, of course it should be the one you showed me. Nice catch!

**Comment by mkmkme** [comment]
Should I squash all this or can it be done via github webui?

**Comment by enjoy-binbin** [comment]
> Should I squash all this or can it be done via github webui?

no need, no worry about the incremental commits, we will eventually use the squash merge

**Review by enjoy-binbin — APPROVED** [review]

**Pull request merged**